projects/interfaces/buffer_test.go

```diff
@@ -0,0 +1,51 @@
+package OurBuffer
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// If you make a buffer named b containing some bytes, calling b.Bytes() returns the same bytes you created it with.
+func TestBufferBytesReturnsInitialBytes(t *testing.T) {
+	inputBytes := []byte("Hello, World!")
+	b := NewBuffer(inputBytes)
+
+	bytes := b.Bytes()
+
+	require.Equal(t, len(inputBytes), len(bytes))
```

Comment:
In all of these tests I would assert on the _contents_ of these slices (probably converted to `string`s for better error messages), rather than just the length.